app/components/search-facet-taxonomy.js

```diff
@@ -86,6 +86,7 @@ export default Ember.Component.extend(Analytics, {
                     })
                 });
                 this._expandMany(items);
```

Comment:
There is definitely the opportunity for conflict between the expanding done by this line and your expandDefault. 
For example, I tested clicking a 3rd level subject and then refreshed the page. Instead of showing the selected subject it only expanded out two levels. Swapped back to develop and it expanded out for my clicked subject. 

This may require a product for to answer if they possibly want one or the other to occur (e.g., expand the clicked items or expand the default items) or if they always want both, in which case a non-toggle function will need to be written that always expands. 

If they want only one or the other, you can probably check if anything is in items and if not, expandDefault.